frame/nfts/src/types.rs

```diff
@@ -40,6 +43,12 @@ pub(super) type ItemTipOf<T, I = ()> = ItemTip<
 	BalanceOf<T, I>,
 >;
 
+pub trait Incrementable {
+	fn increment(&self) -> Self;
```

Comment:
```suggestion
	fn saturating_increment(&self) -> Self;
```

Reply:
Let's do that during the final PR to master